test-cypress/cypress/e2e/1-getting-started/login.cy.js

```diff
@@ -0,0 +1,37 @@
+/// <reference types="cypress" />
+describe('example to-do app', () => {
+    beforeEach(() => {
+      cy.visit('http://localhost:8080/courses/timetable')
+    })
+  
+    // it('displays two todo items by default', () => {
```

Comment:
are we going to use this commented code? Otherwise it would be good to remove it.